members/members/adiati98.ts

```diff
@@ -42,8 +43,8 @@ export const adiati98: MemberObject = {
 		// { type: 'youtube', channelId: 'yourChannelId' }, OR { type: 'youtube', customUrl: 'https://www.youtube.com/c/yourCustomUrl' },
 		{ type: 'polywork', username: 'adiatiayu' },
 		// { type: 'medium', username: 'yourUserName' },
-		{ type: 'hashnode', username: 'ayuadiati' },
+		{ type: 'hashnode', username: 'adiatiayu' },
 		// { type: 'website', url: 'https://virtualcoffee.io', title: 'Title of link' },
 	],
-	badges: [],
+	badges: [Hacktoberfest2022],
```

Comment:
Hey @adiati98! This needs to be a string (wrapped in `'`) - that should fix it up:

```suggestion
	badges: ['Hacktoberfest2022'],
```

Reply:
Thanks, @danieltott!
I've pushed the changes just now 😄